src/Calendar/__tests__/Navigation.jsx

```diff
@@ -508,13 +508,15 @@ describe('Navigation', () => {
     const date = new Date(2017, 0, 1);
     const label = 'Custom label';
     const view = 'month';
+    const locale = 'en-US';
```

Comment:
This might be silly, but can you change this to 
```suggestion
    const locale = 'de-DE';
```

? We're using that in other tests because en-US is the default that getUserLocale would return, so we're not really testing if proper `locale` is passed.